### PR TITLE
feat(withdraw): support withdrawing rewards to specified address

### PR DESCRIPTION
**Pull request by lxcmyf**
**What does this PR do?**
support withdrawing rewards to specified address, close [TIP-624](https://github.com/tronprotocol/tips/issues/624)

**Why are these changes required?**

**This PR has been tested by:**
- Unit Tests
- Manual Testing

**Follow up**

**Extra details**

### CODE REVIEW

**Review by Mgarcia1996 — APPROVED** [review]

File: actuator/src/main/java/org/tron/core/actuator/WithdrawBalanceToActuator.java [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      throw new ContractValidateException(
+          ACCOUNT_EXCEPTION_STR + readableOwnerAddress
+              + "] is a guard representative and is not allowed to withdraw Balance");
+    }
+
+    long latestWithdrawTime = accountCapsule.getLatestWithdrawTime();
+    long now = dynamicStore.getLatestBlockHeaderTimestamp();
+    long witnessAllowanceFrozenTime = dynamicStore.getWitnessAllowanceFrozenTime() * FROZEN_PERIOD;
+
+    if (now - latestWithdrawTime < witnessAllowanceFrozenTime) {
```
Comment (lurais) [inline actuator/src/main/java/org/tron/core/actuator/WithdrawBalanceToActuator.java:142]: Why should we add the frozen time limit?

File: actuator/src/main/java/org/tron/core/actuator/WithdrawBalanceToActuator.java [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+    AccountCapsule accountCapsule = accountStore.get(ownerAddress);
+    String readableOwnerAddress = StringUtil.createReadableString(ownerAddress);
+    if (accountCapsule == null) {
+      throw new ContractValidateException(
+          ACCOUNT_EXCEPTION_STR + readableOwnerAddress + NOT_EXIST_STR);
+    }
+
+    byte[] receiverAddress = withdrawBalanceToContract.getReceiverAddress().toByteArray();
+    if (!DecodeUtil.addressValid(receiverAddress)) {
```
Comment (lurais) [inline actuator/src/main/java/org/tron/core/actuator/WithdrawBalanceToActuator.java:118]: What if the ownerAddress is equal to the receiverAddress?

**Comment by Mgarcia1996** [comment]
Disculpa soy nuevo en esto y la verdad no entiendo bien en general que es
lo que tengo que hacer y mis activos aún siguen

File: actuator/src/main/java/org/tron/core/actuator/WithdrawBalanceToActuator.java (hunk 1 above)
Comment (lxcmyf) [reply actuator/src/main/java/org/tron/core/actuator/WithdrawBalanceToActuator.java:142]: Please refer to the historical code implementation.

Comment (Mgarcia1996) [reply actuator/src/main/java/org/tron/core/actuator/WithdrawBalanceToActuator.java:142]: Disculpa y eso dónde se consulta

**Comment by sergei17081985** [comment]
title: Allow claimed vote reward sent to appointed address 2200152318956128
author: 1000000
discussions-to: <https://github.com/tronprotocol/tips/issues/624>
status: Draft
type: Standards Track 
category : Core
created: 2024.02.22

**Comment by sergei17081985** [comment]
Переводы себе на карту альфабанк

**Pull request closed without merging**